content/develop/connect/insight/_index.md

```diff
@@ -100,6 +100,10 @@ Advanced command line interface with intelligent command auto-complete and compl
 
   <img src="images/Workbench_TimeSeries.png">
 
+Here's an example of auto-completion for the Redis Query Engine:
```

Comment:
```suggestion
Here's an example of auto-completion for Redis Query Engine:
```

Reply:
Advanced, schema-aware auto-complete for [Redis Query Engine](https://redis.io/docs/latest/develop/interact/search-and-query/?utm_source=redisinsight&utm_medium=main&utm_campaign=release_notes) is available in Workbench.
Auto-complete brings faster query building with context-sensitive suggestions that recognize indexes, schemas, and fields based on your current query. Start typing any [Redis Query Engine](https://redis.io/docs/latest/commands/?group=search) command in to try this feature.

Reply:
See next merge.